cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
Whatever the unit is for the RST API, the field name should reflect that. For the timeout, we made it milliseconds.